doc/source/user_guide/pystan2_refitting.ipynb

```diff
@@ -35,7 +35,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For the example we will use a linear regression."
+    "For the example, we will use linear regression."
```

Comment:
I think it should be "a linear regression" as linear regression is like a kind of models and here we are using a specific linear regression

Reply:
Should I change it to "For the example, we will use a linear regression model." or "linear regression approach"?
Or it should be just "linear regression"?

Reply:
I _think_ both "a linear regression model" and "a linear regression" are fine, even "a linear regression line" probably :thinking: